pkg/printer/printer.go

```diff
@@ -0,0 +1,13 @@
+package printer
+
+type Printer interface {
+	HighlightText(s string) string
+	BigLettersWithColor(colorStr, text string) (string, error)
```

Comment:
This is definitely nitpicking, but I feel `BitTextWithColor` reads better. Because its similar to the other method `HighlightText` and also the pterm library calls it `DefaultBigText` 😁